orca-echo/src/main/groovy/com/netflix/spinnaker/orca/echo/pipeline/ManualJudgmentStage.groovy

```diff
@@ -155,9 +156,25 @@ class ManualJudgmentStage implements StageDefinitionBuilder, RestartableStage, A
           application: stage.execution.application
         ),
         additionalContext: [
+          stageName: stage.name,
+          stageId: stage.refId,
+          restrictExecutionDuringTimeWindow: stage.context.restrictExecutionDuringTimeWindow,
           instructions: stage.context.instructions ?: ""
         ]
-      ))
+      )
+
+      if (stage.execution instanceof Pipeline) {
```

Comment:
Good idea. I've updated this to send the entire execution instead of cherry-picking those two attributes.